documentation/network-configuration-guide.html

```diff
@@ -0,0 +1,403 @@
+            <code>2001:0db8:0000:0000:0000:2b3e:3d80:61a1</code>).
+        </p>
+
+        <section id="ipv6-nic">
+
+            <h3>Network Interfaces</h3>
+
+            <p>
+                By default, Openfire will bind to all network interfaces, including IPv6 interfaces. When Openfire has
+                been configured to bind to a specific interface, this may exclude IPv6 connectivity. The configuration
+                can be found in the <code>openfire.xml</code> configuration file.
+            </p>
+
+            <fieldset>
+                <legend>Example <code>openfire.xml</code> configuration for XMPP binding interface</legend>
+                <pre><code>
+&lt;!-- Network settings. By default, Openfire will bind to all network interfaces.
+     Alternatively, you can specify a specific network interfaces that the server
+     will listen on. For example, 127.0.0.1. This setting is generally only useful
+     on multi-homed servers. --&gt;
+&lt;!--
+&lt;network&gt;
+    &lt;interface&gt;&lt;/interface&gt;
+&lt;/network&gt;
+--&gt;
+                </code></pre>
+            </fieldset>
+            
+            <p>
+                The <code>openfire.xml</code> configuration file can contain a similar element to define on what interface
+                the Admin Console is accessible.
+            </p>
+
+            <fieldset>
+                <legend>Example <code>openfire.xml</code> configuration for admin console binding interface</legend>
+                <pre><code>
+&lt;adminConsole&gt;
+    &lt;!-- Disable either port by setting the value to -1 --&gt;
+    &lt;port>9090&lt;/port&gt;
+    &lt;securePort>9091&lt;/securePort&gt;
+    &lt;interface>127.0.0.1&lt;/interface&gt;
+&lt;/adminConsole>
+                </code></pre>
+            </fieldset>
+
+            <p>
+                Both can be used to bind to an IPv6 network interface.
+            </p>
+        </section>
+
+        <section id="ipv6-dns">
+
+            <h3>DNS records</h3>
+
+            <p>
+                If the server on which Openfire is running can be connected to using IPv6, it will have an IPv6 address.
+                When that's the case, then a DNS AAAA record must be created to map the hostname of the server to its
+                address. This is analogous to an A record, that maps an IPv4 address to the hostname.
+            </p>
+
+        </section>
+
+        <section id="ipv6-java">
+
+            <h3>Java Networking Configuration</h3>
+
+            <p>
+                Openfire is an application that runs in a Java virtual machine. Openfire's IPv6 support is primarily
+                controlled by standard <a href="https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/net/doc-files/net-properties.html">Java Networking Properties</a>.
+                The following properties are typically provided. Please refer to the documentation of your version of
+                Java for details.
```

Comment:
Should this be a bit more explicit, given the stuff below on what the launch script does?

Example:
```suggestion
                Java for details on available settings and how to control them (e.g. using <code>JAVA_TOOL_OPTS</code>.
```

Reply:
Do we even support that property? I'm not sure if we even have a generic way of applying these settings across all platforms that we support. To avoid confusion, I think we best leave this out for now.